cli/cmd/options.go

```diff
@@ -66,10 +66,11 @@ func makeInstallUpgradeFlags(defaults *l5dcharts.Values) ([]flag.Flag, *pflag.Fl
 					// individual flags can override the HA defaults.  This means that the HA flag must appear
 					// before the individual flags in the slice passed to flags.ApplyIfSet.
 					values.ControllerReplicas = haValues.ControllerReplicas
-					values.Global.Proxy.Cores = haValues.Global.Proxy.Cores
 					values.Global.Proxy.Resources.CPU.Request = haValues.Global.Proxy.Resources.CPU.Request
 					values.Global.Proxy.Resources.Memory.Request = haValues.Global.Proxy.Resources.Memory.Request
-					values.Global.Proxy.Resources.CPU.Limit = haValues.Global.Proxy.Resources.CPU.Limit
+					// NOTE: CPU Limits are not currently set by default HA charts.
+					//values.Global.Proxy.Cores = haValues.Global.Proxy.Cores
+					//values.Global.Proxy.Resources.CPU.Limit = haValues.Global.Proxy.Resources.CPU.Limit
```

Comment:
wondering what difference this makes as we are storing empty values of these variables even without this change? :thinking:

Reply:
AIUI, this prevents us from overwriting values set by `--proxy-cpu-limit` when it is set before `--ha`